src/NativeScript/CMakeLists.txt

```diff
@@ -58,6 +58,9 @@ set(HEADER_FILES
     ObjC/Inheritance/ObjCClassBuilder.h
     ObjC/Inheritance/ObjCExtend.h
     ObjC/Inheritance/ObjCTypeScriptExtend.h
+    ObjC/Unmanaged/UnmanagedType.h
```

Comment:
I know it isn't obvious, but we try to keep these lists sorted, so that merge conflicts wouldn't happen as often if two or more users add files. If you wish to change it, the correct order here should be:

```
ObjC/TNSDictionaryAdapter.h
ObjC/Unmanaged/UnmanagedInstance.h
ObjC/Unmanaged/UnmanagedPrototype.h
ObjC/Unmanaged/UnmanagedType.h
Runtime/JSWeakRefConstructor.h
```